apple/internal/testing/apple_test_bundle_support.bzl

```diff
@@ -353,10 +353,22 @@ def _apple_test_bundle_impl(*, ctx, product_type):
         bundle_loader = None
 
     extra_linkopts = [
+        "-Xlinker",
+        "-needed_framework",
+        "-Xlinker",
+        "XCTest",
         "-framework",
         "XCTest",
         "-bundle",
     ]
+
+    if platform_prerequisites.uses_swift:
+        extra_linkopts.extend([
+            "-Xlinker",
+            "-needed-lXCTestSwiftSupport",
+            "-lXCTestSwiftSupport",
```

Comment:
Do we need an Xcode version check? Or is this valid for our minimum supported Xcode versions?